rollup.config.js

```diff
@@ -86,11 +131,42 @@ export default [
       },
     ],
     plugins: [
+      applyVueCompat,
```

Comment:
looks like this plugin is always used, so it can be part of plugins?

Reply:
not always, read line 142~170